primitives/src/locktime/absolute.rs

```diff
@@ -438,10 +451,11 @@ mod tests {
     }
 
     #[test]
-    fn lock_time_from_invalid_hex_should_err() {
-        let hex = "0xzb93";
-        let result = LockTime::from_hex(hex);
-        assert!(result.is_err());
+    fn invalid_should_err() {
```

Comment:
Why the rename? if anything, I'd rename it to `invalid_hex` since "should err" is implied by "invalid".

Reply:
The meaning was all invalid locktime values should error.  I have renamed it as suggested and made a new test for the non hex values.